lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -110,9 +141,6 @@ public function getName()
     public function toArray()
```

Comment:
I think now we can completely remove the toArray as it will just take the parent one.